<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushPutAccountBalance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Put Balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAddDrink">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Post Drink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetAccount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetDrink">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Drink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushFavoritsMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Favorits Drink</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textMenu">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:28pt;&quot;&gt;dfjkodf&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textInformation">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>100</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMainMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPostAccount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Post Account</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textInsertName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:22pt;&quot;&gt;Name: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPutAccount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Put Account</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textInsertBalance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:22pt;&quot;&gt;Balance: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetAllAccount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get All Account</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textId">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:22pt;&quot;&gt;ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDeleteAccount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetAccountBalance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Balance by ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPutRecipe">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Put Recipe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPutDrink">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Put Drink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDeleteDrink">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Drink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetAllDrinks">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Get All Drinks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetRecipe">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Recipe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPostRecipe">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Post Recipe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDeleteRecipe">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Recipe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetAllRecipe">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get All Recipes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAddFavorits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Favorits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusDeleteFavorist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Favorits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetIngredient">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Ingredient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPostIngredient">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Post Ingredient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPutIngredient">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Put Ingredient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetAllIngredient">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get All Ingredient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDeleteIngredient">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Ingredient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetContainer">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Container</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPostContainer">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Post Container</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPutContainer">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Put Container</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDeleteContainer">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Container</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGetAllContainer">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get All Container</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
